api/src/main/java/marquez/common/models/JobVersionId.java

```diff
@@ -19,4 +19,9 @@ public class JobVersionId {
   @NonNull NamespaceName namespace;
   @NonNull JobName name;
   @NonNull UUID version;
+
+  public static JobVersionId of(
```

Comment:
why not `new` if we already have that constructor?

Reply:
We do have such a convention for model classes and I kept constant with it.

Reply:
Yeah, it's a factory method we have for our models as convention throughout the code base. Thanks @pawel-big-lebowski for keeping things constant ❤️